stl/src/cond.cpp

```diff
@@ -23,8 +23,8 @@ struct _Cnd_internal_imp_t { // condition variable implementation for ConcRT
 };
 
 static_assert(sizeof(_Cnd_internal_imp_t) <= _Cnd_internal_imp_size, "incorrect _Cnd_internal_imp_size");
-static_assert(std::alignment_of<_Cnd_internal_imp_t>::value <= _Cnd_internal_imp_alignment,
-    "incorrect _Cnd_internal_imp_alignment");
+static_assert(
+    std::alignment_of_v<_Cnd_internal_imp_t> <= _Cnd_internal_imp_alignment, "incorrect _Cnd_internal_imp_alignment");

```

Comment:
(Insert "what year is it??!?" meme here.)

Could we please use `alignof` instead of the ought-to-be-deprecated `std::alignment_of_v`?

Reply:
I forgot that the Core Language existed.